backend/lambdas/custom_resources/validate_vpce_subnets.py

```diff
@@ -0,0 +1,40 @@
+import boto3
+from crhelper import CfnResource
+from decorators import with_logging
+
+helper = CfnResource(json_logging=False, log_level="DEBUG", boto_level="CRITICAL")
+
+ec2_client = boto3.client("ec2")
+
+
+@with_logging
+@helper.create
+@helper.update
+def create(event, context):
+    props = event.get("ResourceProperties", None)
```

Comment:
nit: if your code _needs_ these values to be set, then it's probably not worth using the `.get` method of `dict` because that implies you're fine with the default return value of `None` which it doesn't look like we are. In the event that one of these props isn't set, then presumably we want the function to error and rollback

I have seen this approach done in one of the other custom resources but I don't think it's correct, especially given this is a validation scenario. I've marked this comment as a "nit" because in reality, what would probably happen is that the function will error anyways when you try to call a method on `None`, but it will be slightly harder to debug

Reply:
Yes, i agree. As you said there are other CRs that do it in this project so i did it here to follow the others and be consistant.